scss-0.7.0/environment/tools/functions/_asset-urls.scss

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
Unnecessary blank line, please remove.

Reply:
This is in the FEL. Whats your thoughts on me adding this to the FEL and we catch it next time?

Reply:
Ah ok. Sure, if you get it sorted.

Reply:
You know what.... lets just get it sorted now...